tests/cli/test_deployment_cli.py

```diff
@@ -372,6 +440,35 @@ def test_parsing_rrule_schedule_json(self, patch_import, tmp_path):
             deployment.schedule.rrule
             == "DTSTART:20220910T110000\nRRULE:FREQ=HOURLY;BYDAY=MO,TU,WE,TH,FR,SA;BYHOUR=9,10,11,12,13,14,15,16,17"
         )
+        assert deployment.schedule.timezone == "America/New_York"
+
+    def test_parsing_rrule_timezone_overrides_if_passed_explicitly(
+        self, patch_import, tmp_path
+    ):
+        invoke_and_assert(
+            [
+                "deployment",
+                "build",
+                "fake-path.py:fn",
+                "-n",
+                "TEST",
+                "-o",
+                str(tmp_path / "test.yaml"),
+                "--rrule",
+                '{"rrule": "DTSTART:20220910T110000\\nRRULE:FREQ=HOURLY;BYDAY=MO,TU,WE,TH,FR,SA;BYHOUR=9,10,11,12,13,14,15,16,17", "timezone": "America/New_York"}',
+                "--timezone",
+                "Europe/Berlin",
```

Comment:
sanity check: is this expected that the user can set the timezone on both rrule and separately? I'd expect if the one on rrule dict is set, no extra --timezone flag should be provided, but no strong opinions on what the right behavior should be here

Reply:
I do think it's important to be able to override the timezone explicitly here even if one is provided, `RRule` strings can be complex and specify multiple timezones for each of its components

Reply:
I see, some users could be confused which one is the one that will get used in the end, but this is probably sth we can tackle with docs, thx for clarifying